client/src/elements/schema-editor/schema-editor-array.html

```diff
@@ -5,7 +5,7 @@
   <legend>
     ${schema.title}
   </legend>
-  <div if.bind="data" repeat.for="entry of data" class="schema-editor-array__entry">
+  <div if.bind="data" repeat.for="entry of data" class="schema-editor-array__entry" style="z-index: ${data.length - $index}">
```

Comment:
i do not really understand what you mean with this. but I guess it's fine right?